digiroad2-oracle/src/main/resources/db/migration/V236__create_LengthOfRoadAxis_properties.sql

```diff
@@ -0,0 +1,205 @@
+-- NEW ASSET TYPE
+INSERT INTO ASSET_TYPE(ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (450, 'Pituussuuntaiset tiemerkinnät', 'linear', 'db_migration_v236');
+
+-- LOCALIZED_STRING VALUE
+DECLARE
+    foundItem VARCHAR2(256);
+BEGIN
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Asetusnumero';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Asetusnumero', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Kaistannumero';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Kaistannumero', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Kaistantyyppi';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Kaistantyyppi', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Suhteellinen sijainti';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Suhteellinen sijainti', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Merkinnän materiaali';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Merkinnän materiaali', 'db_migration_v236', SYSDATE);
+
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Merkinnän pituus';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Merkinnän pituus', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Merkinnän leveys';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Merkinnän leveys', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Profiilimerkintä';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Profiilimerkintä', 'db_migration_v236', SYSDATE);
+    END IF;
+
+    SELECT COUNT(*) INTO foundItem FROM LOCALIZED_STRING WHERE VALUE_FI = 'Lisätieto';
+    IF foundItem = 0 THEN
+        INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+        VALUES (primary_key_seq.nextval, 'Lisätieto', 'db_migration_v236', SYSDATE);
+
+    END IF;
```

Comment:
For example, in this case, they already exist:
SELECT * FROM LOCALIZED_STRING WHERE VALUE_FI = 'Lisätieto';

For me doesn't need to be there, only will generate confusion in the future, because you never know when was really inserted.